<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InregistrareAngajat</class>
 <widget class="QDialog" name="InregistrareAngajat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 242, 241);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_prenume">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prenume">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Prenume</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_parola">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nume">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nume">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Nume</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_parola">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Parolă</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../PROIECT/logo_mic.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inregistrare">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(9, 52, 111);
color: white;
font-size: 20px;
border-style: outset;
border-radius: 20px;
font: Times New Roman;</string>
   </property>
   <property name="text">
    <string>Înregistrare</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inregistrare_2">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(9, 52, 111);
color: white;
font-size: 15px;
border-style: outset;
border-radius: 20px;
font: Times New Roman;</string>
   </property>
   <property name="text">
    <string>Aveți deja cont?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
